src/main/java/scmspain/karyon/restrouter/exception/HandlerNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package scmspain.karyon.restrouter.exception;
+
+/**
+ * Created by borja.vazquez on 25/9/15.
+ */
```

Comment:
We decided before not to use creation comments as we think they are totally devoid of value